read_config.h

```diff
@@ -0,0 +1,20 @@
+#include <iostream>
+#include <fstream>
+
+namespace config_reader {
+
+std::ifstream file("config.txt");
```

Comment:
This function definition should be in its own projects Cpp file and not the .h file.